test/e2e/config/helm.yaml

```diff
@@ -0,0 +1,144 @@
+managementClusterName: caaph-e2e
+
+images:
+  - name: ${MANAGER_IMAGE}
+    loadBehavior: mustLoad
+  - name: registry.k8s.io/cluster-api/cluster-api-controller:v1.6.0
```

Comment:
Should we bump CAPI to v1.6.1 in general (or is that for a separate PR)?

Reply:
Let's leave that for a separate PR (since we'd have to manually test all over again).